emData/generate_PR.py

```diff
@@ -74,7 +74,12 @@ def writeSourceModuleInstance(module, wiresFileName):
     strSource += "#pragma HLS inline off\n"
     strSource += "#pragma HLS interface register port=bx_o\n"
     for i in range(TProjCount):
+        strSource += "#ifdef  __VITIS_HLS__\n"
```

Comment:
Why is this #ifdef needed? It just seems to control whether function get_mem() is referred to using a pointer or not. Why should this change depending on whether Vitis is in use?